gradle.properties

```diff
@@ -15,7 +15,7 @@ cloth_version=11.0.99
 modmenu_version=7.0.1
 
 minecraft_version=1.20.1
-yarn_mappings=1.20.1+build.5
+yarn_mappings=1.20.1+build.1
```

Comment:
Was this an unnecessary commit?

Reply:
For whatever reason, my gradle has trouble with yarn mappings loading on build.2-5 so I changed it to build.1. Must have accidentally committed it, can change.

Reply:
Yeah I can't build w/ `1.20.1+build.5` 🤷

![image](https://github.com/BVengo/simple-shulker-preview/assets/4142480/bea5cbf3-22a1-452c-a8e6-96598a1f140d)

Very peculiar

Reply:
That's bizarre - I'll make sure to test it myself when I get a chance.

Reply:
Alright I've done some testing and 5 seems to work without any issues on a raw clone of your repo - not sure what's happened on your end! Since both versions work, I'll accept this change anyway just to keep the PR simple.